### PR TITLE
Fix history task message

### PR DESCRIPTION
**Pull request by Lainow**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | !33795

Removed the code passage that added a group if the delete right was deactivated, as this was causing problems when cracking the escaldae spot (the delete right check is done in JavaScript).

### CODE REVIEW

**Comment by Rom1-B** [comment]
Can you add some tests, because it seems to me that this is the 3rd recent PR that modifies this part of the code.

**Review by Rom1-B — CHANGES_REQUESTED** [review]
Can you add some tests, because it seems to me that this is the 3rd recent PR that modifies this part of the code.

File: tests/autoload.php [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    [
+        GLPI_ROOT . '/plugins',
+        GLPI_ROOT . '/tests/fixtures/plugins',
+    ]
+);
+
+define('TU_USER', '_test_user');
+define('TU_PASS', 'PhpUnit_4');
+
+global $CFG_GLPI, $GLPI_CACHE;
```
Comment (RomainLvr) [inline tests/autoload.php:54]:
```suggestion
/**
 * @var array $CFG_GLPI
 * @var \Psr\SimpleCache\CacheInterface $GLPI_CACHE
 */
global $CFG_GLPI, $GLPI_CACHE;
```

File: tests/units/TaskMessage.php [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        ];
+    }
+
+    /**
+     * @dataProvider dataTestEscalationTaskGroup
+     */
+    public function testEscalationTaskGroup($expected, $inputs)
+    {
+        $ticket = new Ticket();
+        $ticket->update($inputs);
```
Comment (RomainLvr) [inline tests/units/TaskMessage.php:119]:
```suggestion
        $this->assertTrue($ticket->update($inputs));
```

**Review by RomainLvr — CHANGES_REQUESTED** [review]

**Review by RomainLvr — APPROVED** [review]

File: tests/units/TaskMessageTest.php [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with Escalade. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2015-2023 by Escalade plugin team.
+ * @license   GPLv2 https://www.gnu.org/licenses/gpl-2.0.html
+ * @link      https://github.com/pluginsGLPI/escalade
+ * -------------------------------------------------------------------------
+ */
+
+namespace GlpiPlugin\Scim\Tests\Units;
```
Comment (Rom1-B) [inline tests/units/TaskMessageTest.php:31]:
```suggestion
namespace GlpiPlugin\Escalade\Tests\Units;
```

File: tests/units/TaskMessageTest.php [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    public function testPluginReactivated()
+    {
+        $this->login();
+        PluginEscaladeConfig::loadInSession();
+        $_SESSION['glpilanguage'] = 'en_GB';
+        $plugins = new Plugin();
+        $plugins->getFromDBbyDir('escalade');
+        $this->assertTrue(Plugin::isPluginActive('escalade'));
+    }
+    protected function dataTestEscalationTaskGroup(): array
```
Comment (Rom1-B) [inline tests/units/TaskMessageTest.php:51]: Not used?

**Review by Rom1-B — CHANGES_REQUESTED** [review]

**Review by Rom1-B — APPROVED** [review]
🎉 we have tests!

**Review by AdrienClairembault — APPROVED** [review]
Great work 👍 

Two minor things to fix and OK for merge

File: tests/bootstrap.php [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+ * You should have received a copy of the GNU General Public License
+ * along with Escalade. If not, see <http://www.gnu.org/licenses/>.
+ * -------------------------------------------------------------------------
+ * @copyright Copyright (C) 2015-2023 by Escalade plugin team.
+ * @license   GPLv2 https://www.gnu.org/licenses/gpl-2.0.html
+ * @link      https://github.com/pluginsGLPI/escalade
+ * -------------------------------------------------------------------------
+ */
+
+global $CFG_GLPI, $PLUGIN_HOOKS;
```
[the remark this answers is not in the document]
Comment (Lainow) [reply tests/bootstrap.php]: It is used because when I remove it the tests fail.

**Comment by AdrienClairembault** [comment]
Log path still seems invalid: 

image

File: tests/bootstrap.php [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+define('GLPI_ROOT', __DIR__ . '/../../../');
```
Comment (Rom1-B) [inline tests/bootstrap.php:33]: To test:
```suggestion
define('GLPI_ROOT', dirname(__DIR__, 3));
```

**Comment by stonebuzz** [comment]
@Lainow 

can you adapt CHANGELOG.md ?

File: CHANGELOG.md [hunk 7]
```diff
@@ -7,6 +7,14 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
...
 ## [unrelease] -
 
+### Fixed
+
+- Correction of the task message generated during escalation
+
+### Added
+
+- Configuring transiflex as a CI
+
```
Comment (stonebuzz) [inline CHANGELOG.md:17]: ? XD

**Review by stonebuzz — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 7 above)
Comment (Rom1-B) [reply CHANGELOG.md:17]: You can remove this part, as it doesn't affect the use of the plugin, it's just internal stuff.

**Review by stonebuzz — APPROVED** [review]

**Pull request merged**